src/main/java/com/minecolonies/entity/ai/citizen/lumberjack/Tree.java

```diff
@@ -105,18 +115,23 @@ else if(block instanceof BlockNewLog)
             }
 
             location = getBaseLog(world, log);
+            topLog = getTopLog(world, log);
+
             woodBlocks = new LinkedList<>();
+            addAndSearch(world, location);
+
             checkTree(world, getTopLog(world, log));
             stumpLocations = new ArrayList<>();
+            woodBlocks = new LinkedList<>();
```

Comment:
Why are we reseting woodBlocks, and if that was intended, would clear be better? (<- legitimate question it could be worse)